reccmp/isledecomp/compare/db.py

```diff
@@ -44,12 +61,12 @@ def name(self) -> Optional[str]:
         return self.options.get("name")
 
     @property
-    def size(self) -> Optional[int]:
-        return self.options.get("size")
+    def size(self) -> int:
+        return self.options.get("size", 0)
```

Comment:
I'd add a docstring explaining that zero is returned when no size attributes is present